tutorial/input_output.py

```diff
@@ -0,0 +1,234 @@
+from .common import Question, Quiz
+
+
+class StringOutput(Quiz):
+    def __init__(self, title=""):
+        q1 = Question(
+            question="Can the function <code>print()</code> be used on strings only?",
+            options={
+                "False": "Correct! It can be used on any Python object.",
+                "True:": "Wrong! Try it in a cell below.",
+            },
+            correct_answer="False",
+            shuffle=True,
+        )
+
+        q2 = Question(
+            question="What does a <code>f</code> before a string do? E.g <code>f'Hello {name}'</code>",
+            options={
+                "It formats the string as a float": "Wrong! Try it in a cell below.",
+                "It changes the color of the string": "Wrong! Try it in a cell below.",
+                "It allows you to insert variables into the string": "Correct! In the example above, it will print the value of the <code>name</code> variable, if this is defined.",
```

Comment:
```suggestion
                "It allows inserting variables into a string": "Correct! In the example above, it will print the value of the <code>name</code> variable, if it is defined.",
```